docs/reference/search/search.asciidoc

```diff
@@ -244,6 +268,7 @@ by their respective types in the response. Defaults to `true`.
 (Optional, boolean)
 If `true`, returns document version as part of a hit. Defaults to `false`.
 
+[role="child_attributes"]
```

Comment:
For my own knowledge, what does this do?

Reply:
This adds some CSS styling to better render collapsible sections for nested properties.

Because we don't want the styling on all collapsible sections, we have to add this attribute
so our build attaches the `child_attributes` CSS class to the section.

Here's an example of how it looks when rendered:

<img width="750" alt="Screen Shot 2020-06-09 at 11 58 00 AM" src="https://user-images.githubusercontent.com/40268737/84171234-97d66980-aa48-11ea-8f6a-cac1a72eeccc.png">

Reply:
👍 thanks!